packages/commons/src/Deque.AxeCore.Commons.csproj

```diff
@@ -49,4 +49,18 @@
   <ItemGroup>
     <EmbeddedResource Include="node_modules/axe-core/axe.min.js" Link="Resources/axe.min.js" />
   </ItemGroup>
+
+  <!-- Running this using BeforeTagets="Restore" doesn't work if we run "dotnet restore" from solution folder. 
+       Running it before CollectPackageReferences, will work in both vs & command line. But, this task will be executed in both build & restore multiple times on multi-target project.
+       So, skipping this task if the copied package-lock.json file is same as the source file, to support incremental builds.
+  -->
+  <Target Name="DownloadAxe" Inputs="package-lock.json" Outputs="node_modules/package-lock.json" BeforeTargets="CollectPackageReferences">
+    <Exec Command="npm ci" />
+    <!-- Copying this file to support incremental builds -->
+    <Copy SourceFiles="package-lock.json" DestinationFolder="node_modules" />
+  </Target>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Resources/sampleFile.txt" />
```

Comment:
This is only used for tests; it should be in the test assembly, not the src one

Reply:
(the file itself as well as any csproj registration for it)

Reply:
Thanks for clarifying. I've moved the file to the test assembly as well as the csproj registration